changelogs/fragments/depreciation.yml

```
@@ -0,0 +1,7 @@
+---
+minor_changes:
```

There is a heading in the changelog for this:

```suggestion
deprecated_features:
```